packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1255,4 +1262,40 @@ void main() {
     expect(find.text('C'), findsOneWidget);
     expect(find.text('D'), findsNothing);
   });
+
+  testWidgets('Correct custom expanded padding', (WidgetTester tester) async {
+    const Key firstPanelKey = Key('firstPanelKey');
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SingleChildScrollView(
```

Comment:
Might be a silly question, but why a `SingleChildScrollView`?

Reply:
Just used it as all other tests did it the same way and from my pov I see no disadvantage here.
Do you see any downsides or have a better alternative for embedding?

Reply:
I was just wondering if it was possible to remove it. It reduces the test case and makes it less complex